config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1280,7 +1280,7 @@ presubmits:
         - '--node-test-args=--service-feature-gates="UserNamespacesSupport=true,ProcMountType=true" --feature-gates="UserNamespacesSupport=true,ProcMountType=true" --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
         - --node-tests=true
         - --provider=gce
-        - --test_args=--nodes=8 --focus="\[NodeFeature:ProcMountType\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
+        - --test_args=--nodes=1 --focus="\[NodeFeature:ProcMountType\]|\[NodeFeature:UserNamespacesSupport\]"
```

Comment:
should we explicitly query for Serial tag here? It is not ideal to run non-serial tests (if there will be any) in the serial job

Reply:
according to https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/127484/pull-kubernetes-node-crio-cgrpv2-userns-e2e-serial/1836871735580823552 there's only 16 tests. maybe not ideal to have the only option to run the non serial jobs serially, but the impact is not that wide and I figured it'd be worth saving the complexity in the job definitions

Reply:
actually only 2 are non serial jobs. 3 are serial tests, and the remaining 11 are setup/teardown

Reply:
we need at least to add a comment about it. Whenever we have non-standard test description we need to clarify why they are non-standard